EIPS/eip-2612.md

```diff
@@ -38,7 +38,7 @@ While it may be tempting to introduce `*_by_signature` counterparts for every ER
 Three new functions are added to the ERC-20 ABI:
 ```sol
 function permit(address owner, address spender, uint value, uint deadline, uint8 v, bytes32 r, bytes32 s) external
-function nonces(address owner) external view returns (uint)
+function nonces(address owner) external view returns (uint256)
```

Comment:
While I am in support of spelling out it. the uint is an alias of uint256 for now. if you like to change line 41, consider also change everywhere else e.g. line 40, for consistency

Reply:
Leave to riply